source/expressions.tex

```diff
@@ -1821,7 +1820,7 @@
 any attributes are treated as
 \grammarterm{attribute-specifier-seq} in \grammarterm{lambda-expression}.
 \begin{note}
-Such ambiguous cases cannot have valid semantics
+Such ambiguous cases would not have valid semantics
 because the constraint expression would not have type \keyword{bool}.
```

Comment:
```suggestion
Such ambiguous cases do not have valid semantics,
because the constraint expression needs to have type \keyword{bool}.
```

Reply:
Let's leave this as is, since I'm not sure if "needs" is actually safe.